### PR TITLE
Normalize relevant times

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/ext/patient.rb [hunk 1]
```diff
@@ -138,6 +138,41 @@ def ethnicity
...
     end
 
+    # Iterates through each data element to add
+    # 1. A Relevant DateTime if a Relevant Period is specified
+    # 2. A Relevant Period if a Relevant DateTime is specified
+    # This is used to normalize for eCQM calculations that may use one or the other
+    # The denormalize_as_datetime flag is used by the "denormalize_date_times" to return the record to the original state
+    def normalize_date_times
+      qdmPatient.dataElements.each do |de|
+        next unless de.respond_to?(:relevantDatetime) && de.respond_to?(:relevantPeriod)
```
Comment (lmd59) [inline lib/ext/patient.rb:148]: This checks that it responds to both relevantDatetime and relevantPeriod, but it doesn't check that exactly one is null/undefined. Do we have to worry about it ever having both be null or both be non-null/defined?

**Review by lmd59 — APPROVED** [review]
